.github/workflows/docReport.yml

```diff
@@ -40,11 +40,11 @@ jobs:
         env:
           NODE_OPTIONS: --max_old_space_size=8192
 
-      - name: Create Docs Report on current
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/current' }}
+      - name: Create Docs Report on current or next
+        if: ${{ github.event_name == 'push' && (github.ref == 'refs/heads/current' || github.ref == 'refs/heads/next') }}
```

Comment:
Not sure if the `event_name` is still push when a PR is merged. Does anyone know?

Reply:
Seems fair to assume so, esp. since there doesn't seem to be a `merge` event.

Reply:
It might be trial and error. I will merge and check